doc/architecture-design.md

```diff
@@ -494,7 +494,7 @@ Set the route that best suits your business needs in the local configuration `co
 
 [Back to top](#Table-of-contents)
 
-## Consumer
+## Consumer1
```

Comment:
why add 1？

Reply:
@moonming - If you check here- https://github.com/apache/apisix/blob/master/doc/architecture-design.md#table-of-contents , the correct link for Consumer is showing as https://github.com/apache/apisix/blob/master/doc/architecture-design.md#consumer-1

Reply:
> > > @Firstsawyou Facing issue while adding a semantic PR title..Need to know the process in detail..As this commit is already pushed, not getting from where to start. It will be great if you could help me with some steps :) ..Here is my mail id- [radhikajadhav014@gmail.com](mailto:radhikajadhav014@gmail.com)
> > 
> > 
> > You can click the `Edit` button to the right of the "PR" title to change the `PR` title. Add `docs:` before your title `Corrected the link`, like this: `dosc: Corrected the link`
> 
> Its done, thanks @Firstsawyou
> But how to start the checks again so that sematic PR check will pass the test?

It still looks like it failed the semantic test. Try to modify it like this: `docs(architecture-design): corrected the link`. The C letter becomes lowercase.

Reply:
@96RadhikaJadhav 

I tried this link and the cause of this problem is the title of `Consumer` on line `382` (https://github.com/apache/apisix/blame/master/doc/architecture-design.md#L382) and ` The `Consumer` on line 497` is duplicated (https://github.com/apache/apisix/blame/master/doc/architecture-design.md#L497).
There is no need to add 1 here. We only need to modify the content of line `23` to `- [**Consumer**](#consumer-1)`  .
We also need to update the Chinese documents.

Reply:
@Firstsawyou  I'm getting error while pushing new changes now -->
! [rejected]        Corrected-The-Link -> Corrected-The-Link (non-fast-forward)
error: failed to push some refs to 'https://github.com/96RadhikaJadhav/apisix.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
> @Firstsawyou I'm getting error while pushing new changes now -->
> ! [rejected] Corrected-The-Link -> Corrected-The-Link (non-fast-forward)
> error: failed to push some refs to 'https://github.com/96RadhikaJadhav/apisix.git'
> hint: Updates were rejected because the tip of your current branch is behind
> hint: its remote counterpart. Integrate the remote changes (e.g.
> hint: 'git pull ...') before pushing again.
> hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Before you `git push`, you may need to do `git pull`.

Reply:
I tried but it's not working :(